source/_blog_pagination.erb

```diff
@@ -0,0 +1,20 @@
+<% pagination_context = current_page.metadata[:locals] %>
+
+<footer class="blog-pagination">
+  <nav class="blog-pagination__container">
+    <% if pagination_context["prev_page"] %>
+      <%= link_to "Previous page", pagination_context["prev_page"].url, class: 'blog-pagination__link' %>
+    <% else %>
+      <span class="blog-pagination__link-disabled">Previous page</span>
```

Comment:
Not having the link at all if you can't click it looks much better I think, so I've done that.